Source/WebCore/platform/network/curl/CurlMultipartHandle.cpp

```diff
@@ -348,7 +299,7 @@ bool CurlMultipartHandle::parseHeadersIfPossible()
         if (name.isEmpty())
             break;
 
-        m_headers.append(name.toString() + ": " + value + "\r\n");
+        m_headers.append(makeString(name.toString(), ": ", value, "\r\n"));
```

Comment:
Do you need `.toString()` here? Can you give just `name` here?

Reply:
It it fine to give just `name`.

Reply:
Fixed